packages/tests/shadydom/event-path.html

```diff
@@ -930,6 +937,22 @@
       w.dispatchEvent(new Event('foo'));
       assert.isTrue(heardEvent);
     });
+
+    test('events dispatched to unassigned slottables bubble through the tree above them', () => {
```

Comment:
Let's include a test like the example in this issue: https://github.com/Polymer/lit-element/issues/658.

Reply:
I still need to do this.

Reply:
I added another test that I think is closer to what's happening in that bug. I think the reason I was struggling to come up with a scenario that worked for this without including LitElement itself in the tests was that LitElement actually schedules its first render asynchronously after the first `connectedCallback` but I was under the impression that it was sync during `connectedCallback`. The test I added includes a parent and child, where the parent creates an empty shadow root for itself in its constructor and the child dispatches the event in its constructor (tighter timing than the example in the bug), but I simplified by making the parent never actually fill the shadow root with any content. In LitElement, this happens late enough that I think it doesn't strike me as useful to include in the test: the `connectedCallback` dispatching the event happens sync inside an `innerHTML` set in the test (which is all we care about observing) so there's no need to actually make the child appear in the real tree, AFAICT.